compareTree.py

```diff
@@ -1,211 +1,140 @@
-import os, glob, re, csv, datetime # Imports python modules
+import os
+import glob
+import re
+import csv
+import datetime  # Imports python modules
+import createTree
+from configparser import ConfigParser
 from datetime import tzinfo
```

Comment:
This line can go. Was for a previous attempt at getting the date format correct.